components/skus/browser/rs/lib/Cargo.toml

```diff
@@ -22,7 +22,7 @@ tracing = { version = "0.1", default-features = false }
 bigdecimal = { version = "0.1", features = ["serde"] }
 serde = { version = "1.0", features = ["derive"] }
 chrono = { version = "0.4", default-features = false, features = ["clock", "serde"] }
-uuid = { version = "0.8", features = ["serde"] }
+uuid = { version = "1", features = ["v4"] }
```

Comment:
Please update `brave/third_party/rust/third_party.toml` as well to keep those in sync.

Reply:
I think I already did unless that's different than `third_party/rust/third_party.toml`? sorry, spammed a bunch of updates in last little bit

Reply:
Ah, yes. Sorry I missed it.

Reply:
this one is fine because `v4` feature was already in uuid v1, but fyi (if you didn't already know) if you do this manually and there are feature changes, those also need to be reflected in BUILD.gn. If a changed feature results in dep changes then it's best to run gnrt to automatically resolve everything